cmd/main_test.go

```diff
@@ -476,3 +478,116 @@ var _ = DescribeTable("GetWorkspacesWithAccess querying for workspaces with acce
 		})
 	})
 })
+
+var _ = Describe("GetUserNamespaces", func() {
+	var e *echo.Echo
+	var req *labels.Requirement
+	var expectedNamespaces []string
+
+	BeforeEach(func() {
+		e = echo.New()
```

Comment:
I am not sure we need to create the echo server for each test case. it can be moved to the former section of the test and applied to all testcases.